tests/bud.bats

```diff
@@ -25,6 +25,23 @@ load helpers
   expect_output --substring "options use-vc"
 }
 
+@test "build with inline RUN --network=host" {
+  run_buildah build $WITH_POLICY_JSON -t source -f $BUDFILES/inline-network/Dockerfile1
+  expect_output --substring "Connecting to google.com"
+  expect_output --substring "index.html"
```

Comment:
Perhaps this should be running `readlink /proc/self/ns/net` both during and after the build and comparing the two.

Reply:
True a better test will be `readlink /proc/self/ns/net`